packages/order-experience-api/src/api/getGdpr.ts

```diff
@@ -0,0 +1,451 @@
+import {
+  AbstractController,
+  Data,
+  ExperienceError,
+  StatusCode,
+  UnknownRequest,
+  ValidatedRequest,
+} from '@verkkokauppa/core'
+import type { Response } from 'express'
+import * as yup from 'yup'
+import type { Order, Subscription } from '@verkkokauppa/order-backend'
+import {
+  getOrdersByUserAdmin,
+  getSubscriptionsByUserAdmin,
+} from '@verkkokauppa/order-backend'
+import { withAuthentication } from '@verkkokauppa/auth-helsinki-profile'
+
+interface Node {
```

Comment:
Should this interface be on different folder/file?

Reply:
Theres no need to always put interfaces to own files. Its good practise but not always needed if the interfaces are small and related only to specific place like in this case i guess.

Reply:
Good point, I will remember that as well.